test/datatypes.test.js

```diff
@@ -39,13 +37,8 @@ describe('MySQL specific datatypes', function () {
 
   it('should fail spectacularly with invalid enum values', function (done) {
     var em = EnumModel.create({animal: 'horse', condition: 'sleepy', mood: 'happy'}, function (err, obj) {
-      assert.ok(!err);
-      EnumModel.findById(obj.id, function (err, found) {
-        assert.ok(!err);
-        assert.equal(found.animal, ''); // MySQL fun.
-        assert.equal(found.animal, 0);
-        done();
-      });
+      assert.ok(err);
```

Comment:
This will pass for any error (including connection failure). Could you please check that the test failed with an expected error? I believe MySQL provides error codes that can be used for that.